CODEOWNERS

```
@@ -1,9 +1,9 @@
 # Global owners
 
-- @cloudflare/wrangler
-  /packages/wrangler/CHANGELOG.md @cloudflare/wrangler-admins
-  /packages/pages-shared/ @cloudflare/pages
-  /packages/wrangler/pages/ @cloudflare/pages
-  /packages/wrangler/src/pages/ @cloudflare/pages
-  /packages/wrangler/src/pages.\* @cloudflare/pages
-  /packages/create-cloudflare/ @jculvey @RamIdeas @petebacondarwin
+* @cloudflare/wrangler
+/packages/wrangler/CHANGELOG.md @cloudflare/wrangler-admins
```

```suggestion
/packages/wrangler/CHANGELOG.md @cloudflare/wrangler-admins
/.changeset/ @cloudflare/wrangler @cloudflare/pages @cloudflare/d1
/fixtures/ @cloudflare/wrangler @cloudflare/pages @cloudflare/d1
```
Just realised without this, each team will be blocked on updating fixtures/changesets

---

Maybe we should make .changeset not owned by anyone?